docs/docs/guides/integrate/login/login-users.mdx

```diff
@@ -0,0 +1,240 @@
+Developers can customize elements such as logos, colors, and messaging to ensure a seamless integration with their application's user interface.
+
+:::info Customization and Branding
+The login page can be changed by customizing different branding aspects and you can define a custom domain for the login (eg, login.acme.com).
+
+By default, the displayed branding is defined [based on the user's domain](/docs/guides/solution-scenarios/domain-discovery). In case you want to show the branding of a specific organization by default, you need to either pass a primary domain scope (`urn:zitadel:iam:org:domain:primary:{domainname}`) with the authorization request, or define the behavior on your Project's settings.
+:::
+
+### Fast account switching
+
+The hosted login page remembers users that have previously authenticated.
+In case a user has used multiple accounts, for example a private account and a work account, to authenticate, then all accounts will be shown on the Account Picker.
+Users can still login with a different user that is not in the list.
+This allows users to quickly switch between users and provide a better user experience.
+
+:::info
+This behavior can be changed with the authorization request. Please refer to our [guide](/guides/integrate/login/oidc/login-users).
+:::
+
+### Self-service for users
+
+ZITADEL's hosted login page offers [many self-service flows](/docs/concepts/features/selfservice) that allow users to setup authentication methods or recover their login information.
+Developers use the self-service functionalities to reduce manual tasks and improve user experience.
+Key features include:
+
+### Password reset
+
+Unauthenticated users can request a password reset after providing the loginname during the login flow.
+
+- User selects reset password
+- An email will be sent to the verified email address
+- User opens a link and has to provide a new password
+
+#### Prompt users to setup multifactor authentication
+
+Users are automatically prompted to provide a second factor, when
+
+- Instance or organization [login policy](/concepts/structure/policies#login-policy) is set
+- Requested by the client
+- A multi-factor is setup for the user
+
+When a multi-factor is required, but not set-up, then the user is requested to set-up an additional factor.
+
+:::info Disabling multifactor prompt
+You can disable the prompt, in case multifactor authentication is not enforced by setting the [**Multifactor Init Lifetime**](/docs/guides/manage/console/instance-settings#login-lifetimes) to 0.
+:::
+
+#### Enroll passkeys
+
+Users can select a button to initiate passwordless login or use a fall-back method (ie. login with username/password), if available.
+
+The passwordless login flow follows the FIDO2 / WebAuthN standard.
+Briefly explained the following happens:
+
+- User selects button
```

Comment:
What is meant with "user selects button"?

Reply:
removed for now. we probably want to add some video or media on the page.